src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class MainApp {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Random figure = new Random();

```

Comment:
```suggestion
        Random random = new Random();

```
that's not quite a figure